src/opencage.rs

```diff
@@ -23,21 +23,44 @@
 //! // "Carrer de Calatrava, 68, 08017 Barcelona, Spain"
 //! println!("{:?}", res.unwrap());
 //! ```
+use crate::chrono::naive::serde::ts_seconds::deserialize as from_ts;
+use crate::chrono::NaiveDateTime;
+use crate::Deserialize;
+use crate::Point;
+use crate::UA_STRING;
+use crate::{Client, HeaderMap, HeaderValue, USER_AGENT};
+use crate::{Forward, Reverse};
+use failure::Error;
+use num_traits::Float;
+use std::collections::HashMap;
 use std::sync::{Arc, Mutex};
 
-use super::num_traits::Float;
-use std::collections::HashMap;
+mod string_or_int {
+    use crate::{Deserialize, Deserializer};
 
-use super::reqwest;
-use super::Deserialize;
-use super::UA_STRING;
-use super::{header, Client};
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<String, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        #[derive(Deserialize)]
+        #[serde(untagged)]
+        enum StringOrInt {
+            String(String),
+            Int(i32),
+        }
 
-use super::Point;
-use super::{Forward, Reverse};
+        match StringOrInt::deserialize(deserializer)? {
+            StringOrInt::String(s) => Ok(s),
+            StringOrInt::Int(i) => Ok(i.to_string()),
+        }
+    }
+}
 
 // OpenCage has a custom rate-limit header, indicating remaining calls
-header! { (XRatelimitRemaining, "X-RateLimit-Remaining") => [i32] }
+// header! { (XRatelimitRemaining, "X-RateLimit-Remaining") => [i32] }
+static XRL: &'static str = "x-ratelimit-remaining";
```

Comment:
```suggestion
static XRL: &str = "x-ratelimit-remaining";
```

IIRC, static references have a `'static` lifetime by default